.circleci/config.yml

```diff
@@ -152,6 +152,10 @@ jobs:
       - run:
           name: Install Apps plugin
           command: make deploy
+      - run:
+          name: Start hello-world App
+          command: make hello_world
+          background: true
```

Comment:
Why is `background` used here? What happens if the cypress tests are run before the `hello_world` app started?

Reply:
@hanzei If `background` is not used here, the step for starting the `hello_world` app never finishes, because it just listens and never exits. Using `background` may not be the right solution but it works. Any thoughts on how to properly accomplish this?

> What happens if the cypress tests are run before the `hello_world` app started?

The app install fails, and subsequently the e2e test fails

Reply:
Does `make hello_world&` do the trick?

Reply:
@hanzei I haven't tried yet but I think that might make the CI step exit since the foreground will run to completion. Or are you saying that starting the `hello_world` app should be within the step that runs the tests?